sdks/python/apache_beam/runners/common.py

```diff
@@ -565,9 +588,14 @@ def process_outputs(self, windowed_input_element, results):
     then dispatched to the appropriate indexed output.
     """
     if results is None:
+      # TODO[BEAM-3937]:Remove if block after output feature released.
```

Comment:
<!--new_thread; commit:c89749a0d77350dfbd2686db975aa087bafb8f1b; resolved:0-->
Use the form `"# TODO(BEAM-1111): My comment"`.